dj/sql/parsing/ast.py

```diff
@@ -710,6 +711,7 @@ def use_alias_as_name(self) -> "Column":
     def is_compiled(self):
         return self._is_compiled or (self.table and self._type)
 
+    # flake8: noqa
```

Comment:
This should be moved to the end of the next line to just noqa for this function. Otherwise it will noqa for all lines in the file after this line.

Reply:
Done! (flake8 thinks it's too complex 😭 )